File: dataset_reduzido/avatar/Python/Code/atcoder_ABC152_C.py
import numpy as np
N = int ( input ( ) )
n = [ int ( x ) for x in input ( ).split ( ) ]
n = np.array ( n )
a = 1
pos = 0
for i in range ( 1 , N ) :
    if n [ pos ] > n [ i ] :
        a += 1
        pos = i
print ( a )

File: output/LLaMa_repaired_1/avatar/Python/Java/atcoder_ABC152_C.java
import java.util.Scanner;

public class atcoder_ABC152_C {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter the number of elements: ");
        int N = scanner.nextInt();
        scanner.nextLine(); // Consume newline left-over

        System.out.print("Enter the array of elements: ");
        String[] input = scanner.nextLine().split(" ");
        int[] n = new int[N];

        for (int i = 0; i < N; i++) {
            if (i < input.length) {
                n[i] = Integer.parseInt(input[i]);
            }
        }

        int a = 1;
        int pos = 0;
        for (int i = 1; i < N; i++) {
            if (n[pos] < n[i]) {
                a += 1;
                pos = i;
            }
        }

        System.out.println(a);
    }
}